<launch>
    <node name="object_detection_service_node" pkg="yolo_object_detection" type="object_detection_service.py" output="screen"/> 
    <node name="object_extraction_service_node" pkg="pcl_object_extraction" type="pcl_object_extraction_service_node" output="screen"/>
    <node name="mesh_creation_service_node" pkg="pcl_object_extraction" type="mesh_creation_service_node" output="screen"/>
</launch>